### ISSUE TITLE
[BUG] OpenVINO/Qwen3.8-27B-int4-ov incoherent on GPU and CPU (ignored_scope null on GatedDeltaNet)

### ISSUE THREAD

**Issue by SergiioB** [openvinotoolkit/openvino.genai#4467]
## Bug Description

Official Hub IR [`OpenVINO/Qwen3.8-27B-int4-ov`](https://huggingface.co/OpenVINO/Qwen3.8-27B-int4-ov) loads on OpenVINO GenAI 2026.5 and then emits incoherent greedy output. Same string on **GPU and CPU**. This is the Qwen3.8-27B instance of the ignored-scope GDN bug from #3870 (closed on 3.6).

`openvino_config.json` on that card:

- `ignored_scope: null` (top-level and `lm_model`)
- INT4_ASYM `ratio=1.0` so GatedDeltaNet / `linear_attn` is quantized

## Steps to Reproduce

1. `hf download OpenVINO/Qwen3.8-27B-int4-ov`
2. Load with `openvino_genai.VLMPipeline(path, "GPU.0")` (or `"CPU"`)
3. Official-style generate with a real image (HF ai2d demo JPEG) and `generation_config=GenerationConfig(max_new_tokens=32)` **as a keyword**

## Actual Behavior

GPU.0:

```
")-+&-&)&/-,'/!&*%!'.33).)'03*#*,"
```

CPU (251 s): **identical**. Dummy zero image → all `!` (token id 0). Fresh GPU JIT cache does not change it.

## Expected Behavior

Coherent English, as with Hub INT8 (`OpenVINO/Qwen3.8-27B-int8-ov`) and with a mixed INT4 that ignores `linear_attn`.

## Fix that works on Arc Pro B70

Export BF16 `Qwen/Qwen3.8-27B` (Transformers 5.2.0), then:

```python
from nncf import CompressWeightsMode, IgnoredScope, compress_weights
compressed = compress_weights(
    language_model,
    mode=CompressWeightsMode.INT4_ASYM,
    group_size=128,
    ratio=1.0,
    ignored_scope=IgnoredScope(patterns=[".*linear_attn.*"]),
)
```

NNCF: 73% INT4, 22% float (GDN), 1 INT8 layer. Language bin 21 GB vs 14 GB all-INT4.

Working IR + card: https://huggingface.co/SergiioB/Qwen3.8-27B-int4-gdn8-ov

On the same B70, greedy n=128, official ai2d image: **13.89 tok/s QUALITY_PASS** (`fire` in the image). Hub INT8: 13.09 PASS. Hub INT4: FAIL.

## Environment

- GPU: Intel Arc Pro B70 32 GB, 150 W cap
- OpenVINO / GenAI: 2026.5 (Python venv)
- Host: Linux, 60 GB RAM
- Also: Cascadia 0.2.3 stock `ov-genai` (bundled OpenVINO 2026.2) SIGSEGVs in `Tokenizer::setup_tokenizer` on this VLM IR. Separate from the logits bug.

Please re-export the Hub INT4 with GDN ignored, or un-list it until then. Community mirrors (`circulus/*`, `Morteza89/*`) copy `ignored_scope: null` as well.

**Comment by as-suvorov** [comment]
@SergiioB Thank you for issue and analysis. As potential fix is on NNCF side, could you please open issue for https://github.com/openvinotoolkit/nncf
